disperser/cmd/apiserver/flags/flags.go

```diff
@@ -47,7 +47,7 @@ var (
 		Name:     common.PrefixFlag(FlagPrefix, "bls-operator-state-retriever"),
 		Usage:    "Address of the BLS Operator State Retriever",
 		Required: true,
-		EnvVar:   common.PrefixEnvVar(envVarPrefix, "BLS_OPERATOR_STATE_RETRIVER"),
+		EnvVar:   common.PrefixEnvVar(envVarPrefix, "BLS_OPERATOR_STATE_RETRIEVER"),
```

Comment:
Such change will break the users who are running this

Reply:
currently it also breaks new user, who expect a consistent spelling.  What's the best way to proceed to consistency?

Reply:
I wouldn't say it "break" new users, although it may create some confusing with wrong spelling.

Reply:
Agreed, please take over this PR and fix it the right way.  As an operator, I'd like consistency for downstream usage.